### PR TITLE
Add --eip4844 devnet flag

### PR DESCRIPTION
**Pull request by Inphi**
Configures the genesis and network parameters of the protodanksharding devnet,

### CODE REVIEW

**Comment by Inphi** [comment]
Gonna merge now even though the bootnodes preconfiguration doesn't quite work as bootnodes are currently iffy on the devnet.
Users will be able to connect to the network by manually providing a `StaticNodes` config.

File: cmd/geth/consolecmd.go [hunk 1]
```diff
@@ -143,6 +143,8 @@ func remoteConsole(ctx *cli.Context) error {
 				path = filepath.Join(path, "sepolia")
 			} else if ctx.GlobalBool(utils.KilnFlag.Name) {
 				path = filepath.Join(path, "kiln")
+			} else if ctx.GlobalBool(utils.KilnFlag.Name) {
```
Comment (protolambda) [inline cmd/geth/consolecmd.go:146]: This is a bug?

Comment (Inphi) [reply cmd/geth/consolecmd.go:146]: yeah, it's a bug. It should be the eip4844 flag

Comment (Inphi) [reply cmd/geth/consolecmd.go:146]: though this has been fixed since.

**Pull request merged**